src/main/java/com/kryptnostic/rhizome/mapstores/cassandra/SetProxyBackedCassandraMapStore.java

```diff
@@ -102,7 +102,7 @@ public V load( K key ) {
         List<Row> execute = session.execute( LOAD_ALL_KEYS.bind() ).all();
         Set<K> set = Sets.newHashSetWithExpectedSize( execute.size() );
         for ( Row row : execute ) {
-            K key = keyMapper.toKey( row.getString( CassandraQueryConstants.VALUE_RESULT_COLUMN_NAME ) );
+            K key = keyMapper.toKey( row.getString( SetProxy.KEY_COLUMN_NAME ) );
```

Comment:
Looks good because the comment for `Row.getString` indicates `@param name the name of the column to retrieve.`